src/config/ConfigManager.cpp

```diff
@@ -1953,15 +1954,16 @@ std::optional<std::string> CConfigManager::handleBind(const std::string& command
     // bind[fl]=SUPER,G,exec,dmenu_run <args>
 
     // flags
-    bool       locked       = false;
-    bool       release      = false;
-    bool       repeat       = false;
-    bool       mouse        = false;
-    bool       nonConsuming = false;
-    bool       transparent  = false;
-    bool       ignoreMods   = false;
-    bool       multiKey     = false;
-    const auto BINDARGS     = command.substr(4);
+    bool       locked          = false;
+    bool       release         = false;
+    bool       repeat          = false;
+    bool       mouse           = false;
+    bool       nonConsuming    = false;
+    bool       transparent     = false;
+    bool       ignoreMods      = false;
+    bool       multiKey        = false;
+    bool       has_description = false;
```

Comment:
no snake :skull:

hasDescription

Reply:
python muscle mem, it seems.
fixed in commit 05a1e22ed93afefb84b19bec05ed66d3de2e568b